src/modules/tree.ts

```diff
@@ -294,7 +294,7 @@ export default {
     requiredArgs: 'account',
   },
   test: {
-    description: 'Run unit tests',
+    description: 'Run your VTEX app tests',
```

Comment:
*nit*: I think you can keep *app unit tests* once there isn't a specific format to our tests